schemas/pioneer-core/demographic-survey/demographic-survey.1.schema.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$comment": "mozilla-ion/ion-core-addon#182 - Collect demographic information",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "pioneer_core",
+    "bq_metadata_format": "pioneer",
+    "bq_table": "demographic_survey_v1"
+  },
+  "properties": {
+    "age": {
+      "additionalProperties": false,
+      "properties": {
+        "19_24": {
```

Comment:
Note that BigQuery doesn't support columns that start with a number, so they will be prefixed with an underscore.

Reply:
Thanks for mentioning this. How will the column name look like in this case? `age._19_24`?

Reply:
Yep, the sink will add this before inserting. You can preview the columns in the PR bot (the message disappeared though?)

Reply:
Ah neat, looks like I was able to find it [here](https://github.com/mozilla-services/mozilla-pipeline-schemas/commit/733aa3f91397ca998cc98d6443e910385c15f050#comments). Thanks! Looks good to me :)